notifier/orb.yaml

```diff
@@ -1,12 +1,56 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4.7.0
+
 executors:
   alpine-stable:
     docker:
     - image: alpine:latest
     resource_class: small
 
+commands:
+  slack-on-deploy:
```

Comment:
I think this should be `slack-on-fail`, right? This will only be triggering in cases of failure (i.e. `event: fail` is the hard-coded setting here).

Reply:
Adding @shaundialpad since we have conflicting opinions. Sorry Shaun for changing my mind, but, I think that Jon is right here. The essence of this command is to send an alert when a job fails. We are indeed going to use it only for deploys on production, but, this was only our decision and not something that the command has.

Do you agree?

Reply:
yep definitely `slack-on-fail` :P